components/Actuators/HighLevel/shooterLogic.py

```diff
@@ -97,7 +98,8 @@ def isShooterUpToSpeed(self):
                 and abs(self.shooterMotor2Encoder.getVelocity() - shootSpeed2) <= abs(self.shootTolerance))
         rumble  = 0
         if atSpeed and not self.isAutonomous:
-            rumble = .3
+            rumble = 1
+        logging.error("1: %s vs %s, 2: %s vs %s", shootSpeed1, self.shooterMotor1Encoder.getVelocity(), shootSpeed2, self.shooterMotor2Encoder.getVelocity())
```

Comment:
Won't this log (and do getVelocity()) on every frame? Could this be what's slowing us down?